<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameStartDialog</class>
 <widget class="QDialog" name="GameStartDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>186</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>开始</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item row="6" column="1">
    <widget class="QLabel" name="BlackDepthLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>搜索深度</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLineEdit" name="BlackDepth">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="RedPlayer">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>人类</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>计算机</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QComboBox" name="BlackDiff">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>简单</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>困难</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>自定义</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="RedDepthLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>搜索深度</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="RedLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>红方</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="RedDepth">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="RedDiffLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>难度</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QComboBox" name="BlackPlayer">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>人类</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>计算机</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="BlackDiffLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>难度</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="BlackLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>黑方</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="RedDiff">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>简单</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>困难</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>自定义</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GameStartDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GameStartDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
